cli/command/service/update.go

```diff
@@ -640,6 +640,12 @@ func updatePlacementPreferences(flags *pflag.FlagSet, placement *swarm.Placement
 }
 
 func updateContainerLabels(flags *pflag.FlagSet, field *map[string]string) {
+	if *field != nil && flags.Changed(flagContainerLabelRemove) {
```

Comment:
question: should we test field to?
```go
if field != nil && *field != nil
```

Reply:
Good one, I easily get confused by these 🤔

Reply:
yeah pointer on pointer is mind blowing 😁

Reply:
I'll look for the extra check in a follow-up